includes/html/pages/apps.inc.php

```diff
@@ -640,7 +640,11 @@
 ];
 
 $apps = LibreNMS\Util\ObjectCache::applications()->flatten();
```

Comment:
Probably better to replace this with an actual query or fix the query in "ObjectCache"

https://github.com/librenms/librenms/blob/master/LibreNMS/Util/ObjectCache.php#L51